test/common/http/http1/codec_impl_test.cc

```diff
@@ -678,8 +678,13 @@ TEST_F(Http1ServerConnectionImplTest, ChunkedResponse) {
   response_encoder->encodeHeaders(headers, false);
 
   Buffer::OwnedImpl data("Hello World");
-  response_encoder->encodeData(data, true);
-  EXPECT_EQ("HTTP/1.1 200 OK\r\ntransfer-encoding: chunked\r\n\r\nb\r\nHello World\r\n0\r\n\r\n",
+  response_encoder->encodeData(data, false);
```

Comment:
Split this into a new test. We should have a chunked test with and without trailers.